### PR TITLE
QoL / wording improvements to explore

### PR DESCRIPTION
**Pull request by annoyingpr**
## Briefly describe the feature or bug that this PR addresses.

Updated Explore page to be clearer to users as per this [Slack discussion](https://serotracker.slack.com/archives/C01LF2V72FN/p1619540612032600).

## Please link the Airtable ticket associated with this PR.
https://airtable.com/tbli2lWQHAqBa6ZcI/viwTYqtBDdt3Wt3Yo/rec0Mwk38OwpzEfPQ?blocks=hide

## If applicable, include screenshots of the feature/bugfix introduced by this PR (Please include both desktop and mobile if there is a significant UI change).
image

## Describe the steps you took to test the feature/bugfix introduced by this PR.
Click around locally and check that everything looks accurate!

## Does this PR depend on any recent backend work? If so, please include the PR number from the iit-backend repo and associated Airtable ticket link.
Nope!

## Does this PR require any French translations? Have they been included?
Yes, many - all have been documented in the #translate-fr Slack channel

### CODE REVIEW

File: src/components/sidebar/left-sidebar/TotalStats.tsx [hunk 1]
```diff
@@ -2,66 +2,66 @@ import React, { useContext, useEffect, useState } from "react";
 import { AppContext } from "../../../context";
 import { State, PageState } from "../../../types";
 import Translate from "../../../utils/translate/translateService";
-import './TotalStats.css';
+import "./TotalStats.css";
```
Comment (atatmaja) [inline src/components/sidebar/left-sidebar/TotalStats.tsx:5]: is it worth to add linting checks to our code and have 1 PR where we fix up linting throughout the app? just finding it hard sometimes to parse out real changes from linting related stuff

**Comment by atatmaja** [comment]
Should the first row say "1515 Seroprevalence Studies" instead? That would be more consistent with the grammatical structure we've employed the rest of the summary stats section

**Review by atatmaja — APPROVED** [review]
Code LGTM, but a few comments on wording and linting

File: src/components/sidebar/left-sidebar/TotalStats.tsx (hunk 1 above)
Comment (annoyingpr) [reply src/components/sidebar/left-sidebar/TotalStats.tsx:5]: 100% agree with this - we can configure Git hooks and also add a check in our CI / CD

**Comment by rahularoradfs** [comment]
I prefer the 1515 seroprevalence studies framing!

**Comment by annoyingpr** [comment]
> I prefer the 1515 seroprevalence studies framing!

Lowercase or uppercase?
image
vs.
image

**Comment by rahularoradfs** [comment]
lowercase!

**Pull request merged**